releasebuilder

```diff
@@ -11,7 +11,7 @@ repository_name = sys.argv[3]
 repository_url = sys.argv[4]
 github_token = sys.argv[5]
 
-header = "Content-Type:application/zip"
+header = "Content-Type:application/zip\r\nAuthorization: token {}".format(github_token)
```

Comment:
Great that you made it work. Just one suggestion. You can have several header (-H) parameters in curl request:

https://stackoverflow.com/questions/356705/how-to-send-a-header-using-a-http-request-through-a-curl-call

So I would create a new variable indtead of using "\r\n" in the string i.e.

header = "Content-Type:application/zip"
header2 = "Authorization: token {}".format(github_token)

"curl", "-XPOST", "-H", header, "-H", header2, "--data-binary", ...

See if it works.